### ISSUE TITLE
[FEATURE] - Linear Searching Algorithms (Java)

### ISSUE THREAD

**Issue by TharunBalaji2004** [Rohan-Saini7996/Hacktoberfest2023#3]
Hi there, I have an active leetcode profile(https://leetcode.com/tharunbalaji31/) and proficient in algorithms. I mostly prefer Java and Python. I'm interested to contribute Searching Algorithms - Linear and Binary Search (with TC and SC)

Kindly assign this feature to me 😊

PS: I have starred this repo for future reference

**Comment by saku1331** [comment]
I wish to work on this issue. Kindly assign me this.

**Comment by Rohan-Saini7996** [comment]
@TharunBalaji2004 @saku1331 guys tell me one thing your PR count is showing after successful merge?

**Comment by TharunBalaji2004** [comment]
@Rohan-Saini7996 Yup, PRs opened and closed between Oct 1 - 30 will only be considered. Kindly try to open issues and assign members after October 1 and also tag the issue as _Hacktoberfest2023_

**Comment by Rohan-Saini7996** [comment]
@TharunBalaji2004 okay i will so wait till 1st October

**Comment by TharunBalaji2004** [comment]
Hey @Rohan-Saini7996 Kindly tag the issue as hacktoberfest, and assign to me 😊
